src/Sulu/Bundle/MediaBundle/Tests/Functional/Entity/MediaDataProviderRepositoryTest.php

```diff
@@ -225,7 +223,7 @@ private function createRole($name, $system)
         return $role;
     }
 
-    private function createAccessControl($entityClass, $id, $permissions, Role $role)
+    private function createAccessControl($entityClass, $id, $permissions, Role $role): AccessControl
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking changes